scene/animation/animation_tree.cpp

```diff
@@ -147,7 +147,7 @@ AnimationTree *AnimationNode::get_animation_tree() const {
 }
 
 AnimationNode::NodeTimeInfo AnimationNode::blend_input(int p_input, AnimationMixer::PlaybackInfo p_playback_info, FilterAction p_filter, bool p_sync, bool p_test_only) {
-	ERR_FAIL_INDEX_V(p_input, inputs.size(), NodeTimeInfo());
+	ERR_FAIL_INDEX_V(p_input, (int64_t)inputs.size(), NodeTimeInfo());
```

Comment:
This is odd isn't there a 64 bit test?

Reply:
I can't find any 64 bit variation of the macro, not very sure what you mean. The conversion is to supress compiler warning and be aligned with `Vector`'s `size()` (it does this internally and return size in `int64_t`)